RHEL6 Failure in TestE2E/SEQ/CONFIG/config_global/ConfigPasswdNo

### Version of Singularity:

What version of Singularity are you using? Run:

```
3.5.2
```
### Actual behavior

```
                --- FAIL: TestE2E/SEQ/CONFIG/config_global/ConfigPasswdNo (0.14s)
                    singularitycmd.go:653: Running command "/usr/local/bin/singularity --debug config global --set config passwd no"
                    config.go:379: Running command "/usr/local/bin/singularity --debug exec /tmp/stest.031722543/test.sif grep /etc/passwd /proc/self/mountinfo"
                    config.go:379: 
                        "/usr/local/bin/singularity --debug exec /tmp/stest.031722543/test.sif grep /etc/passwd /proc/self/mountinfo" output:
                        DEBUG   [U=500,P=30668]    persistentPreRunE()           Singularity version: 3.5.2+164-g739eac2
                        DEBUG   [U=500,P=30668]    handleConfDir()               /home/dtrudg/.singularity already exists. Not creating.
                        DEBUG   [U=500,P=30668]    initCacheDir()                Creating cache directory: /tmp/stest.031722543/e2e-imgcache-230879486/cache
                        DEBUG   [U=500,P=30668]    initCacheDir()                Creating cache directory: /tmp/stest.031722543/e2e-imgcache-230879486/cache/library
                        DEBUG   [U=500,P=30668]    updateCacheSubdir()           Caching directory set to /tmp/stest.031722543/e2e-imgcache-230879486/cache/library
                        DEBUG   [U=500,P=30668]    initCacheDir()                Creating cache directory: /tmp/stest.031722543/e2e-imgcache-230879486/cache/oci-tmp
                        DEBUG   [U=500,P=30668]    updateCacheSubdir()           Caching directory set to /tmp/stest.031722543/e2e-imgcache-230879486/cache/oci-tmp
                        DEBUG   [U=500,P=30668]    initCacheDir()                Creating cache directory: /tmp/stest.031722543/e2e-imgcache-230879486/cache/oci
                        DEBUG   [U=500,P=30668]    updateCacheSubdir()           Caching directory set to /tmp/stest.031722543/e2e-imgcache-230879486/cache/oci
                        DEBUG   [U=500,P=30668]    initCacheDir()                Creating cache directory: /tmp/stest.031722543/e2e-imgcache-230879486/cache/net
                        DEBUG   [U=500,P=30668]    updateCacheSubdir()           Caching directory set to /tmp/stest.031722543/e2e-imgcache-230879486/cache/net
                        DEBUG   [U=500,P=30668]    initCacheDir()                Creating cache directory: /tmp/stest.031722543/e2e-imgcache-230879486/cache/shub
                        DEBUG   [U=500,P=30668]    updateCacheSubdir()           Caching directory set to /tmp/stest.031722543/e2e-imgcache-230879486/cache/shub
                        DEBUG   [U=500,P=30668]    initCacheDir()                Creating cache directory: /tmp/stest.031722543/e2e-imgcache-230879486/cache/oras
                        DEBUG   [U=500,P=30668]    updateCacheSubdir()           Caching directory set to /tmp/stest.031722543/e2e-imgcache-230879486/cache/oras
                        DEBUG   [U=500,P=30668]    execStarter()                 Checking for encrypted system partition
                        DEBUG   [U=500,P=30668]    Init()                        Image format detection
                        DEBUG   [U=500,P=30668]    Init()                        Check for sandbox image format
                        DEBUG   [U=500,P=30668]    Init()                        sandbox format initializer returned: not a directory image
                        DEBUG   [U=500,P=30668]    Init()                        Check for sif image format
                        DEBUG   [U=500,P=30668]    Init()                        sif image format detected
                        VERBOSE [U=500,P=30668]    SetContainerEnv()             Not forwarding SINGULARITY_E2E_COVERAGE from user to container environment
                        VERBOSE [U=500,P=30668]    SetContainerEnv()             Not forwarding SINGULARITY_CACHEDIR from user to container environment
                        VERBOSE [U=500,P=30668]    SetContainerEnv()             Not forwarding SINGULARITY_SYPGPDIR from user to container environment
                        VERBOSE [U=500,P=30668]    SetContainerEnv()             HOME=/home/dtrudg
                        DEBUG   [U=500,P=30668]    init()                        Use starter binary /usr/local/libexec/singularity/bin/starter-suid
                        VERBOSE [U=0,P=30668]      print()                       Set messagelevel to: 5
                        VERBOSE [U=0,P=30668]      init()                        Starter initialization
                        DEBUG   [U=0,P=30668]      load_overlay_module()         Trying to load overlay kernel module
                        DEBUG   [U=0,P=30668]      load_overlay_module()         Overlay seems not supported by the kernel
                        DEBUG   [U=0,P=30668]      get_pipe_exec_fd()            PIPE_EXEC_FD value: 7
                        VERBOSE [U=0,P=30668]      is_suid()                     Check if we are running as setuid
                        VERBOSE [U=0,P=30668]      priv_drop()                   Drop root privileges
                        DEBUG   [U=500,P=30668]    init()                        Read engine configuration
                        DEBUG   [U=500,P=30668]    init()                        Wait completion of stage1
                        VERBOSE [U=500,P=30677]    priv_drop()                   Drop root privileges permanently
                        DEBUG   [U=500,P=30677]    set_parent_death_signal()     Set parent death signal to 9
                        VERBOSE [U=500,P=30677]    init()                        Spawn stage 1
                        DEBUG   [U=500,P=30677]    startup()                     singularity runtime engine selected
                        VERBOSE [U=500,P=30677]    startup()                     Execute stage 1
                        DEBUG   [U=500,P=30677]    StageOne()                    Entering stage 1
                        DEBUG   [U=500,P=30677]    prepareAutofs()               Found "/misc" as autofs mount point
                        DEBUG   [U=500,P=30677]    prepareAutofs()               Found "/net" as autofs mount point
                        DEBUG   [U=500,P=30677]    prepareAutofs()               Could not keep file descriptor for bind path /etc/localtime: no mount point
                        DEBUG   [U=500,P=30677]    prepareAutofs()               Could not keep file descriptor for bind path /etc/hosts: no mount point
                        DEBUG   [U=500,P=30677]    prepareAutofs()               Could not keep file descriptor for home directory /home/dtrudg: no mount point
                        DEBUG   [U=500,P=30677]    prepareAutofs()               Could not keep file descriptor for current working directory /home/dtrudg/singularity/e2e: no mount point
                        DEBUG   [U=500,P=30677]    Init()                        Image format detection
                        DEBUG   [U=500,P=30677]    Init()                        Check for sandbox image format
                        DEBUG   [U=500,P=30677]    Init()                        sandbox format initializer returned: not a directory image
                        DEBUG   [U=500,P=30677]    Init()                        Check for sif image format
                        DEBUG   [U=500,P=30677]    Init()                        sif image format detected
                        DEBUG   [U=500,P=30677]    setSessionLayer()             Attempting to use underlay (enable underlay = yes)
                        VERBOSE [U=500,P=30668]    wait_child()                  stage 1 exited with status 0
                        DEBUG   [U=500,P=30668]    cleanup_fd()                  Close file descriptor 4
                        DEBUG   [U=500,P=30668]    init()                        Set child signal mask
                        DEBUG   [U=500,P=30668]    init()                        Create socketpair for master communication channel
                        DEBUG   [U=500,P=30668]    init()                        Create RPC socketpair for communication between stage 2 and RPC server
                        VERBOSE [U=500,P=30668]    priv_escalate()               Get root privileges
                        VERBOSE [U=0,P=30668]      priv_escalate()               Change filesystem uid to 500
                        VERBOSE [U=0,P=30668]      init()                        Spawn master process
                        DEBUG   [U=0,P=30684]      set_parent_death_signal()     Set parent death signal to 9
                        VERBOSE [U=0,P=30684]      create_namespace()            Create mount namespace
                        VERBOSE [U=0,P=30668]      enter_namespace()             Entering in mount namespace
                        DEBUG   [U=0,P=30668]      enter_namespace()             Opening namespace file ns/mnt
                        DEBUG   [U=0,P=30668]      setns()                       Using syscall() wrapped __NR_setns
                        VERBOSE [U=0,P=30668]      priv_drop()                   Drop root privileges
                        VERBOSE [U=0,P=30684]      create_namespace()            Create mount namespace
                        DEBUG   [U=500,P=30668]    startup()                     singularity runtime engine selected
                        VERBOSE [U=500,P=30668]    startup()                     Execute master process
                        DEBUG   [U=0,P=30691]      set_parent_death_signal()     Set parent death signal to 9
                        VERBOSE [U=0,P=30691]      init()                        Spawn RPC server
                        DEBUG   [U=500,P=30668]    setupSessionLayout()          Using Layer system: underlay
                        DEBUG   [U=500,P=30668]    setupUnderlayLayout()         Creating underlay SESSIONDIR layout
                        DEBUG   [U=500,P=30668]    addRootfsMount()              Mount rootfs in read-only mode
                        DEBUG   [U=500,P=30668]    addRootfsMount()              Image type is 4096
                        DEBUG   [U=500,P=30668]    addRootfsMount()              Mounting block [squashfs] image: /tmp/stest.031722543/test.sif
                        DEBUG   [U=500,P=30668]    addKernelMount()              Checking configuration file for 'mount proc'
                        DEBUG   [U=500,P=30668]    addKernelMount()              Adding proc to mount list
                        VERBOSE [U=500,P=30668]    addKernelMount()              Default mount: /proc:/proc
                        DEBUG   [U=500,P=30668]    addKernelMount()              Checking configuration file for 'mount sys'
                        DEBUG   [U=500,P=30668]    addKernelMount()              Adding sysfs to mount list
                        VERBOSE [U=500,P=30668]    addKernelMount()              Default mount: /sys:/sys
                        DEBUG   [U=500,P=30668]    addDevMount()                 Checking configuration file for 'mount dev'
                        DEBUG   [U=500,P=30668]    addDevMount()                 Adding dev to mount list
                        VERBOSE [U=500,P=30668]    addDevMount()                 Default mount: /dev:/dev
                        DEBUG   [U=500,P=30668]    addHostMount()                Not mounting host file systems per configuration
                        VERBOSE [U=500,P=30668]    addBindsMount()               Found 'bind path' = /etc/localtime, /etc/localtime
                        VERBOSE [U=500,P=30668]    addBindsMount()               Found 'bind path' = /etc/hosts, /etc/hosts
                        DEBUG   [U=500,P=30668]    addHomeStagingDir()           Staging home directory (/home/dtrudg) at /usr/local/var/singularity/mnt/session/home/dtrudg
                        DEBUG   [U=500,P=30668]    addHomeMount()                Adding home directory mount [/usr/local/var/singularity/mnt/session/home/dtrudg:/home/dtrudg] to list using layer: underlay
                        DEBUG   [U=500,P=30668]    addTmpMount()                 Checking for 'mount tmp' in configuration file
                        VERBOSE [U=500,P=30668]    addTmpMount()                 Default mount: /tmp:/tmp
                        VERBOSE [U=500,P=30668]    addTmpMount()                 Default mount: /var/tmp:/var/tmp
                        DEBUG   [U=500,P=30668]    addScratchMount()             Not mounting scratch directory: Not requested
                        DEBUG   [U=500,P=30668]    addCwdMount()                 Using /home/dtrudg/singularity/e2e as current working directory
                        VERBOSE [U=500,P=30668]    addCwdMount()                 Default mount: /home/dtrudg/singularity/e2e: to the container
                        DEBUG   [U=500,P=30668]    addLibsMount()                Checking for 'user bind control' in configuration file
                        DEBUG   [U=500,P=30668]    addFilesMount()               Checking for 'user bind control' in configuration file
                        DEBUG   [U=500,P=30668]    addResolvConfMount()          Adding /etc/resolv.conf to mount list
                        VERBOSE [U=500,P=30668]    addResolvConfMount()          Default mount: /etc/resolv.conf:/etc/resolv.conf
                        DEBUG   [U=500,P=30668]    addHostnameMount()            Skipping hostname mount, not virtualizing UTS namespace on user request
                        DEBUG   [U=500,P=30668]    create()                      Mount all
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting tmpfs to /usr/local/var/singularity/mnt/session
                        DEBUG   [U=0,P=30691]      startup()                     singularity runtime engine selected
                        VERBOSE [U=0,P=30691]      startup()                     Serve RPC requests
                        DEBUG   [U=500,P=30668]    mountImage()                  Mounting loop device /dev/loop0 to /usr/local/var/singularity/mnt/session/rootfs of type squashfs
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/TZ to /usr/local/var/singularity/mnt/session/underlay/etc/TZ
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/alpine-release to /usr/local/var/singularity/mnt/session/underlay/etc/alpine-release
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/apk to /usr/local/var/singularity/mnt/session/underlay/etc/apk
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/conf.d to /usr/local/var/singularity/mnt/session/underlay/etc/conf.d
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/crontabs to /usr/local/var/singularity/mnt/session/underlay/etc/crontabs
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/fstab to /usr/local/var/singularity/mnt/session/underlay/etc/fstab
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/group to /usr/local/var/singularity/mnt/session/underlay/etc/group
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/hostname to /usr/local/var/singularity/mnt/session/underlay/etc/hostname
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/hosts to /usr/local/var/singularity/mnt/session/underlay/etc/hosts
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/init.d to /usr/local/var/singularity/mnt/session/underlay/etc/init.d
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/inittab to /usr/local/var/singularity/mnt/session/underlay/etc/inittab
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/issue to /usr/local/var/singularity/mnt/session/underlay/etc/issue
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/logrotate.d to /usr/local/var/singularity/mnt/session/underlay/etc/logrotate.d
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/modprobe.d to /usr/local/var/singularity/mnt/session/underlay/etc/modprobe.d
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/modules to /usr/local/var/singularity/mnt/session/underlay/etc/modules
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/modules-load.d to /usr/local/var/singularity/mnt/session/underlay/etc/modules-load.d
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/motd to /usr/local/var/singularity/mnt/session/underlay/etc/motd
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/network to /usr/local/var/singularity/mnt/session/underlay/etc/network
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/opt to /usr/local/var/singularity/mnt/session/underlay/etc/opt
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/os-release to /usr/local/var/singularity/mnt/session/underlay/etc/os-release
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/passwd to /usr/local/var/singularity/mnt/session/underlay/etc/passwd
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/periodic to /usr/local/var/singularity/mnt/session/underlay/etc/periodic
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/profile to /usr/local/var/singularity/mnt/session/underlay/etc/profile
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/profile.d to /usr/local/var/singularity/mnt/session/underlay/etc/profile.d
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/protocols to /usr/local/var/singularity/mnt/session/underlay/etc/protocols
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/resolv.conf to /usr/local/var/singularity/mnt/session/underlay/etc/resolv.conf
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/securetty to /usr/local/var/singularity/mnt/session/underlay/etc/securetty
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/services to /usr/local/var/singularity/mnt/session/underlay/etc/services
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/shadow to /usr/local/var/singularity/mnt/session/underlay/etc/shadow
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/shells to /usr/local/var/singularity/mnt/session/underlay/etc/shells
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/ssl to /usr/local/var/singularity/mnt/session/underlay/etc/ssl
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/sysctl.conf to /usr/local/var/singularity/mnt/session/underlay/etc/sysctl.conf
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/sysctl.d to /usr/local/var/singularity/mnt/session/underlay/etc/sysctl.d
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/etc/udhcpd.conf to /usr/local/var/singularity/mnt/session/underlay/etc/udhcpd.conf
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/.singularity.d to /usr/local/var/singularity/mnt/session/underlay/.singularity.d
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/bin to /usr/local/var/singularity/mnt/session/underlay/bin
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/dev to /usr/local/var/singularity/mnt/session/underlay/dev
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/lib to /usr/local/var/singularity/mnt/session/underlay/lib
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/madness to /usr/local/var/singularity/mnt/session/underlay/madness
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/media to /usr/local/var/singularity/mnt/session/underlay/media
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/mnt to /usr/local/var/singularity/mnt/session/underlay/mnt
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/proc to /usr/local/var/singularity/mnt/session/underlay/proc
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/root to /usr/local/var/singularity/mnt/session/underlay/root
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/run to /usr/local/var/singularity/mnt/session/underlay/run
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/sbin to /usr/local/var/singularity/mnt/session/underlay/sbin
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/scif to /usr/local/var/singularity/mnt/session/underlay/scif
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/srv to /usr/local/var/singularity/mnt/session/underlay/srv
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/sys to /usr/local/var/singularity/mnt/session/underlay/sys
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/tmp to /usr/local/var/singularity/mnt/session/underlay/tmp
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/usr to /usr/local/var/singularity/mnt/session/underlay/usr
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/rootfs/var to /usr/local/var/singularity/mnt/session/underlay/var
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/underlay to /usr/local/var/singularity/mnt/session/final
                        DEBUG   [U=500,P=30668]    mountGeneric()                Remounting /usr/local/var/singularity/mnt/session/final
                        DEBUG   [U=500,P=30668]    setPropagationMount()         Set RPC mount propagation flag to SLAVE
                        VERBOSE [U=500,P=30668]    addIdentityMount()            Skipping bind of the host's /etc/passwd
                        VERBOSE [U=500,P=30668]    Group()                       Checking for template group file: /usr/local/var/singularity/mnt/session/rootfs/etc/group
                        VERBOSE [U=500,P=30668]    Group()                       Creating group content
                        DEBUG   [U=500,P=30668]    addIdentityMount()            Adding /etc/group to mount list
                        VERBOSE [U=500,P=30668]    addIdentityMount()            Default mount: /etc/group:/etc/group
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /dev to /usr/local/var/singularity/mnt/session/final/dev
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /etc/localtime to /usr/local/var/singularity/mnt/session/final/etc/localtime
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /etc/hosts to /usr/local/var/singularity/mnt/session/final/etc/hosts
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/etc/singularity/actions to /usr/local/var/singularity/mnt/session/final/.singularity.d/actions
                        DEBUG   [U=500,P=30668]    mountGeneric()                Remounting /usr/local/var/singularity/mnt/session/final/.singularity.d/actions
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /proc to /usr/local/var/singularity/mnt/session/final/proc
                        DEBUG   [U=500,P=30668]    mountGeneric()                Remounting /usr/local/var/singularity/mnt/session/final/proc
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting sysfs to /usr/local/var/singularity/mnt/session/final/sys
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /home/dtrudg to /usr/local/var/singularity/mnt/session/home/dtrudg
                        DEBUG   [U=500,P=30668]    mountGeneric()                Remounting /usr/local/var/singularity/mnt/session/home/dtrudg
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/home/dtrudg to /usr/local/var/singularity/mnt/session/final/home/dtrudg
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /tmp to /usr/local/var/singularity/mnt/session/final/tmp
                        DEBUG   [U=500,P=30668]    mountGeneric()                Remounting /usr/local/var/singularity/mnt/session/final/tmp
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /var/tmp to /usr/local/var/singularity/mnt/session/final/var/tmp
                        DEBUG   [U=500,P=30668]    mountGeneric()                Remounting /usr/local/var/singularity/mnt/session/final/var/tmp
                        VERBOSE [U=500,P=30668]    mountGeneric()                Skipping mount /home/dtrudg/singularity/e2e, /usr/local/var/singularity/mnt/session/final/home/dtrudg/singularity/e2e already mounted
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/etc/resolv.conf to /usr/local/var/singularity/mnt/session/final/etc/resolv.conf
                        DEBUG   [U=500,P=30668]    mountGeneric()                Mounting /usr/local/var/singularity/mnt/session/etc/group to /usr/local/var/singularity/mnt/session/final/etc/group
                        DEBUG   [U=500,P=30668]    create()                      Chroot into /usr/local/var/singularity/mnt/session/final
                        DEBUG   [U=0,P=30691]      Chroot()                      Hold reference to host / directory
                        DEBUG   [U=0,P=30691]      Chroot()                      Called pivot_root on /usr/local/var/singularity/mnt/session/final
                        DEBUG   [U=0,P=30691]      Chroot()                      Change current directory to host / directory
                        DEBUG   [U=0,P=30691]      Chroot()                      Apply slave mount propagation for host / directory
                        DEBUG   [U=0,P=30691]      Chroot()                      Called unmount(/, syscall.MNT_DETACH)
                        DEBUG   [U=0,P=30691]      Chroot()                      Changing directory to / to avoid getpwd issues
                        DEBUG   [U=500,P=30668]    create()                      Chdir into / to avoid errors
                        VERBOSE [U=0,P=30684]      wait_child()                  rpc server exited with status 0
                        DEBUG   [U=0,P=30684]      apply_container_privileges()  Set user ID to 500
                        DEBUG   [U=500,P=30684]    set_parent_death_signal()     Set parent death signal to 9
                        DEBUG   [U=500,P=30684]    startup()                     singularity runtime engine selected
                        VERBOSE [U=500,P=30684]    startup()                     Execute stage 2
                        DEBUG   [U=500,P=30684]    StageTwo()                    Entering stage 2
                        DEBUG   [U=500,P=30668]    PostStartProcess()            Post start process
                        DEBUG   [U=500,P=30668]    Master()                      Child exited with exit status 0
                        265 244 7:0 /etc/passwd /etc/passwd ro,nosuid,nodev,relatime master:44 - squashfs /dev/loop0 ro
                        
                    config.go:379: unexpected success while executing "/usr/local/bin/singularity --debug exec /tmp/stest.031722543/test.sif grep /etc/passwd /proc/self/mountinfo"
                    singularitycmd.go:653: Running command "/usr/local/bin/singularity --debug config global --reset config passwd"
    suite.go:86: Test failed, not removing /tmp/stest.031722543
FAIL
coverage: 88.1% of statements
FAIL	github.com/sylabs/singularity/e2e	297.609s
FAIL
make: *** [e2e-test] Error 1
```



<!-- if this is a feature request, you can ignore this next part -->

### Steps to reproduce this behavior

How can others reproduce this issue/problem?


### What OS/distro are you running

```
RHEL 6
```

--- a/e2e/config/config.go
+++ b/e2e/config/config.go
@@ -91,7 +91,7 @@ func (c configTests) configGlobal(t *testing.T) {
 		},
 		{
 			name:           "ConfigPasswdNo",
-			argv:           []string{c.env.ImagePath, "grep", "/etc/passwd", "/proc/self/mountinfo"},
+			argv:           []string{c.env.ImagePath, "grep", "/etc/passwd.*- tmpfs", "/proc/self/mountinfo"},
 			profile:        e2e.UserProfile,
 			directive:      "config passwd",
 			directiveValue: "no",
@@ -99,7 +99,7 @@ func (c configTests) configGlobal(t *testing.T) {
 		},
 		{
 			name:           "ConfigPasswdYes",
-			argv:           []string{c.env.ImagePath, "grep", "/etc/passwd", "/proc/self/mountinfo"},
+			argv:           []string{c.env.ImagePath, "grep", "/etc/passwd.*- tmpfs", "/proc/self/mountinfo"},
 			profile:        e2e.UserProfile,
 			directive:      "config passwd",
 			directiveValue: "yes",
@@ -107,7 +107,7 @@ func (c configTests) configGlobal(t *testing.T) {
 		},
 		{
 			name:           "ConfigGroupNo",
-			argv:           []string{c.env.ImagePath, "grep", "/etc/group", "/proc/self/mountinfo"},
+			argv:           []string{c.env.ImagePath, "grep", "/etc/group.*- tmpfs", "/proc/self/mountinfo"},
 			profile:        e2e.UserProfile,
 			directive:      "config group",
 			directiveValue: "no",
@@ -115,7 +115,7 @@ func (c configTests) configGlobal(t *testing.T) {
 		},
 		{
 			name:           "ConfigGroupYes",
-			argv:           []string{c.env.ImagePath, "grep", "/etc/group", "/proc/self/mountinfo"},
+			argv:           []string{c.env.ImagePath, "grep", "/etc/group.*- tmpfs", "/proc/self/mountinfo"},
 			profile:        e2e.UserProfile,
 			directive:      "config group",
 			directiveValue: "yes",
@@ -123,7 +123,7 @@ func (c configTests) configGlobal(t *testing.T) {
 		},
 		{
 			name:           "ConfigResolvConfNo",
-			argv:           []string{c.env.ImagePath, "grep", "/etc/resolv.conf", "/proc/self/mountinfo"},
+			argv:           []string{c.env.ImagePath, "grep", "/etc/resolv.conf.*- tmpfs", "/proc/self/mountinfo"},
 			profile:        e2e.UserProfile,
 			directive:      "config resolv_conf",
 			directiveValue: "no",
@@ -131,7 +131,7 @@ func (c configTests) configGlobal(t *testing.T) {
 		},
 		{
 			name:           "ConfigResolvConfYes",
-			argv:           []string{c.env.ImagePath, "grep", "/etc/resolv.conf", "/proc/self/mountinfo"},
+			argv:           []string{c.env.ImagePath, "grep", "/etc/resolv.conf.*- tmpfs", "/proc/self/mountinfo"},
 			profile:        e2e.UserProfile,
 			directive:      "config resolv_conf",
 			directiveValue: "yes",

